media/css/press/press-ie.less

```diff
@@ -0,0 +1,4 @@
+.tooltip {
+    margin-bottom: 20px;
+    max-width: 75%;
+}
```

Comment:
I completely forgot about `novalidate` attribute to have these browsers skip validation and thus, did not test it :-/ Me :disappointed: